package.json

```
@@ -1,78 +1,52 @@
 {
-  "name": "@ngui/auto-complete",
-  "version": "2.0.0",
-  "description": "Angular Input Autocomplete",
-  "license": "MIT",
-  "main": "dist/auto-complete.umd.js",
-  "module": "dist/index.js",
-  "typings": "dist/index.d.ts",
+  "name": "auto-complete",
+  "version": "0.0.0",
```

```suggestion
  "version": "3.0.0",
```
Because it got a lot of breaking changes please make it version 3.0.0

---

It should be fixed. But refer - `auto-complate/projects/auto-complete/package.json` file.